docs/superpowers/plans/2026-04-30-email-channel-allowed-domains.md

```
@@ -0,0 +1,964 @@
+# Supports wildcard subdomains (e.g. "*.example.com").
+# Empty / unset => fail-closed: no inbound email is processed and no email
+# channel can be saved via the form.
+EMAIL_CHANNEL_ALLOWED_DOMAINS = env.list("EMAIL_CHANNEL_ALLOWED_DOMAINS", default=[])
+```
+
+- [ ] **Step 2: Document the env var in .env.example**
+
+In `.env.example`, after line 58 (end of `## Production Email Config` block),
+append:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)